docs/CommandFormatStandards.md

```diff
@@ -59,7 +59,7 @@ Refer to the following guidelines when writing descriptions for syntax segments
 
 - The `summary` is a one line, short description. This appears in the list of sub-segments on online help pages (for example, Zowe root help lists the groups with their short descriptions. Zowe group help lists the actions with their descriptions.
    
-    - Do NOT include punctuation at the end of a summary description for consistency.
+    - Do NOT include punctuation at the end of a summary for consistency.
```

Comment:
This would be more clear if the sentence was restructured a little: "For consistency, do NOT include ... "